### PR TITLE
ex6 - improve readme

### PR DESCRIPTION
**Pull request by FatCaterpillar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
-    
-...so that the tests pass.
\ No newline at end of file
+## Exercise #6: Errors part 1
+
+This time we will explore errors. Let's try to implement error handling using exceptions.
+
+### Exercise description
+
+- Go to [CurrentLesson.kt](src/main/kotlin/com/krzykrucz/elesson/currentlesson/domain/CurrentLesson.kt)
```
Comment (jakubiec) [inline README.md:9]: whenever I click this link in Intelij, it takes me [here](https://github.com/krzykrucz/e-lesson/blob/master/src/main/kotlin/com/krzykrucz/elesson/currentlesson/domain/CurrentLesson.kt) not sure if it's expected

File: README.md [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
+- Implement Lesson Start Time error handling based on exceptions
```
Comment (jakubiec) [inline README.md:10]: `StartingTooEarlyOrTooLate` is ambiguous by definition, shouldn't we have 2 separated errors?

Comment (krzykrucz) [reply README.md:10]: good idea, feel free to create a commit

Comment (FatCaterpillar) [reply README.md:10]: Corrected

**Review by jakubiec — APPROVED** [review]

**Pull request merged**